src/syncPersistCache.js

```diff
@@ -0,0 +1,52 @@
+import { CachePersistor } from 'apollo-cache-persist';
```

Comment:
File name could be `persistCacheSync` to follow nodejs patterns. Having it this way can be confused with sync as action while this is just marker

Reply:
Sure, no problem. Would you like the same convention applied to the new function names? e.g. syncRestore() vs. restoreSync()

Reply:
I care for the end user api to be seamless. Leaving internals for your discretion.

Reply:
This has been resolved. I kept the internals consistent with externals.